scripts/setup/macos/install_viewer.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eu
+
+# Install cbmc-viewer
+
+# Source kani-dependencies to get CBMC_VIEWER_VERSION
+source kani-dependencies
+
+if [ -z "${CBMC_VIEWER_VERSION:-}" ]; then
+  echo "$0: Error: CBMC_VIEWER_VERSION is not specified"
+  exit 1
+fi
+
+set -x
+
+# brew doesn't recognize specific versions of viewer
+brew install aws/tap/cbmc-viewer
```

Comment:
Is this always set at this point? Where is this tap added?

Reply:
`brew install repo` mitigates the need to add a tap but I could make that more explicit if it's safer.